### PR TITLE
Windows: do not resolve files in current working dir if CWD is not explicitly in PATH.

### PR DESCRIPTION
**Pull request by gtardif**
Workaround for https://github.com/golang/go/issues/38736

Signed-off-by: guillaume.tardif <guillaume.tardif@gmail.com>

**What I did**
* duplicate window os/exec.LookPath() and removed resolving current working dir before resolving in PATH

**Related issue**

/test-windows

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: cli/mobycli/lp_unix.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// +build aix darwin dragonfly freebsd linux netbsd openbsd solaris
```
Comment (chris-crone) [inline cli/mobycli/lp_unix.go:1]: Think you can just do a `!windows` here to have the same effect

**Review by chris-crone — APPROVED** [review]
LGTM

File: cli/mobycli/lp_unix.go (hunk 1 above)
Comment (gtardif) [reply cli/mobycli/lp_unix.go]: changed

File: cli/mobycli/exec.go [hunk 2]
```diff
@@ -60,7 +61,16 @@ func mustDelegateToMoby(ctxType string) bool {
 
 // Exec delegates to com.docker.cli if on moby context
 func Exec(root *cobra.Command) {
-	cmd := exec.Command(ComDockerCli, os.Args[1:]...)
+	execBinary := ComDockerCli
+	if runtime.GOOS == "windows" { // workaround for windows issue https://github.com/golang/go/issues/38736
```
Comment (chris-crone) [inline cli/mobycli/exec.go:65]: We don't need this `if`

Comment (gtardif) [reply cli/mobycli/exec.go]: updated

File: scripts/validate/fileheader [hunk 3]
```diff
@@ -24,4 +24,4 @@ fi
```
Comment (budvajda) [inline scripts/validate/fileheader:24]: Duplicate of #

**Review by RonSherfey — COMMENTED** [review]
reviewed changes

**Review by hibitchbye — COMMENTED** [review]
i said what i said

File: cli/mobycli/resolvepath/lp_unix.go
Comment (giolicer) [inline cli/mobycli/resolvepath/lp_unix.go:1]: cli/mobycli/resolvepath/lp_unix.go

**Pull request merged**